winit/src/settings.rs

```diff
@@ -7,13 +7,28 @@ mod platform;
 #[path = "settings/macos.rs"]
 mod platform;
 
+#[cfg(any(
+    target_os = "linux",
+    target_os = "dragonfly",
+    target_os = "freebsd",
+    target_os = "netbsd",
+    target_os = "openbsd",
+))]
```

Comment:
I used this to restrict the target to unix-like systems. This is what winit used until version 0.27. Now they are using a [`free_unix`](https://github.com/rust-windowing/winit/blob/4652d4810548cb48c89964e7b81f8fc59686a4d3/build.rs#L16) which I think includes more systems but `free_unix` is implied by this configuration predicate.
I guess it is okay to use a more restricted configuration predicate than winit?